src/app/first-time-login-dialog/first-time-login-dialog.component.ts

```diff
@@ -23,6 +23,8 @@ export class FirstTimeLoginDialogComponent implements OnInit {
   faSitemap = faSitemap;
   faAdd = faAdd;
   hideFirstTimeLoginDialogControl = new FormControl(false);
+  nextAddOrganisation = false;
```

Comment:
I think an `enum` or type `'join' | 'add'` would be more suitable.

Reply:
I added the string literals